### PR TITLE
Add score test.

### PR DESCRIPTION
**Pull request by zmccaw-insitro**
* Added the option to perform burden analysis using a score test. 

Updated package passes dev checks.

### CODE REVIEW

File: R/utilities.R [hunk 1]
```diff
@@ -1,19 +1,55 @@
...
+  # Full model.
+  fit_full <- stats::glm(
+      y ~ 0 + g + x, family = stats::binomial(link = "logit"))
+  
+  # Null model.
+  fit_null <- stats::glm(
+    y ~ 0 + x, family = stats::binomial(link = "logit"))
+  
+  # LRT.
+  lrt <- stats::anova(fit_null, fit_full)
```
Comment (ctk3b) [inline R/utilities.R:49]: Confirmed offline: this is an incidental fix that we believe will make this correct for cases not directly pertinent to the allelic series work

**Review by ctk3b — APPROVED** [review]
Looking good! :shipit:

File: R/allelic_series_test.R [hunk 2]
```diff
@@ -159,13 +160,6 @@ ASBT <- function(
     return(NA)
   }
 
-  # Degrees of freedom.
-  if (is.vector(agg_geno)) {
-    df <- 1
-  } else {
-    df <- ncol(agg_geno)
-  }
```
[the remark this answers is not in the document]
Comment (zmccaw-insitro) [reply R/allelic_series_test.R:167]: Yes, that's right, and the calculation that required the degrees of freedom has now been packaged into LinearTest and LogisticLRT.

**Comment by zmccaw-insitro** [comment]
Thanks Christoph!

**Pull request merged**